torch/jit/__init__.py

```diff
@@ -49,6 +49,17 @@
 
 from torch.jit._freeze import freeze, optimize_frozen_module
 
+__all__ = [
+    'script', 'trace', 'script_if_tracing', 'is_scripting', 'trace_module', 'fork',
+    'wait', 'export', 'freeze', '_overload', '_overload_method', '_unwrap_optional',
```

Comment:
I'm honestly not sure about this. Do we want these to be public APIs? Also `_unique_state_dict`, `_flatten`, `_get_trace_graph`?